### ISSUE TITLE
change output targets from Nx1 dim to just N

### ISSUE THREAD

**Issue by donovanr** [AllenCellModeling/pytorch_learning_tools#20]
new pytorch doesn't auto-squeeze tensors like that and loss functions e.g. CrossEntropyLoss throw errors.  Also it makes more sense, that singleton dimension is meaningless anyway.
